common/src/main/java/com/microsoft/identity/common/internal/request/MsalBrokerRequestAdapter.java

```diff
@@ -415,6 +417,28 @@ public Bundle getRequestBundleForHello(@NonNull final CommandParameters paramete
         return requestBundle;
     }
 
+    public Intent getRequestIntentForAcquireTokenInteractive(@NonNull final Bundle resultBundle,
```

Comment:
Should there be documentation for this public method?

Reply:
Addressed via https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/918/commits/477877f2d99d6eb7d752f18d941a682b51665c62